### PR TITLE
Fix for bug: #205

### PR DESCRIPTION
**Pull request by maniankara**
examples/,platforms/: Add GIT_EMAIL variable to roles/*.yaml and Readme.md files

1. Now there is a way to set the email from network.yaml
2. Global options have been removed from platform/shared configs git_push role (not to polute the global settings)

### CODE REVIEW

File: platforms/r3-corda/configuration/roles/setup/notary/tasks/main.yaml [hunk 1]
```diff
@@ -102,6 +102,7 @@
     GIT_DIR: "{{ playbook_dir }}/../../../"
     GIT_REPO: "{{ gitops.git_push_url }}"
     GIT_USERNAME: "{{ gitops.username }}"
+    GIT_EMAIL: "{{ gitops.email }}"
```
Comment (sownak) [inline platforms/r3-corda/configuration/roles/setup/notary/tasks/main.yaml]: the "email" key needs to be added in the sample network.yaml files in platforms/<platform>/configuration/samples directory. Also docs/source updated with this info.

Comment (maniankara) [reply platforms/r3-corda/configuration/roles/setup/notary/tasks/main.yaml]: Oh Thanks for pointing out ! Now I have done the changes for the existing 3 platforms

**Comment by maniankara** [comment]
> Missed the branch first time. Please raise the pull request for develop branch and not master.

Done.
Btw, are there some tests/review happening in `develop` branch?

**Comment by sownak** [comment]
> > Missed the branch first time. Please raise the pull request for develop branch and not master.
> 
> Done.
> Btw, are there some tests/review happening in `develop` branch?

Yes, it is manual at the moment and not automatic.

**Pull request merged**